sources/platform/actors/running/store.md

```diff
@@ -3,129 +3,125 @@ title: Actors in store
 
-If you have an [Apify paid plan](https://apify.com/pricing), you will be automatically charged the monthly rental fee at the end of your free trial and you will be able to run the Actor for another month. If you are not subscribed to an Apify plan, you will need to subscribe to one of [Apify's paid plans](https://apify.com/pricing) in order to continue using the Actor after the trial has ended.
+If you have an [Apify paid plan](https://apify.com/pricing), the per month rental fee will be automatically subtracted from your plans' prepaid usage at the end of your free trial and you will be able to run the Actor for another month. If you are not subscribed to an Apify plan, you will need to subscribe to one of [Apify's paid plans](https://apify.com/pricing) in order to continue using the Actor after the trial has ended.
 
-**Can I cancel my Actor rental?**
+##### Can I cancel my Actor rental?
 
-**You can cancel the Actor rental** during your trial or any time after that so that don't get charged when your current Actor rental expires. You can always turn it back on later if you want.
+_You can cancel the Actor rental_ during your trial or any time after that so that don't get charged when your current Actor rental period expires. You can always turn it back on later if you want.
 
-**Where can I see how much I have paid for Actor rental?**
+##### Where can I see how much I have paid for Actor rental?
 
 Since Actor rental fees are paid from prepaid platform usage, these fees conceptually belong under platform usage.
 
-You can find the breakdown of how much you have been charged for rental Actors at the bottom of the **Platform usage** tab in the [Billing](https://console.apify.com/billing) section. The overall sum will also be visible in the usage bar chart and the table at the top of the same tab.
+You can find the breakdown of how much you have been charged for rental Actors within **Actors** tab in **Current period** tab in the [Billing](https://console.apify.com/billing) section.
 
-![Rental Actors billing in Apify Console](./images/store/paid-actors-billing.png)
+![Rental Actors billing in Apify Console](./images/store/billing-paid-actors.png)
 
-### Actors paid per result
+### Pay per result
 
-When you run an Actor that is **paid per result**, you only pay for the successful results that an Actor returns when you run it, and you are not charged for the underlying platform usage. Technically, we validate the **results** against a schema, and you are only charged for the valid items in the [dataset](https://docs.apify.com/platform/storage/dataset) of the Actor run.
+When you run an Actor that is _paid per result_, you pay for the successful results that an Actor returns when you run it, and you are not charged for the underlying platform usage. Technically, we validate the _results_ against a schema, and you are only charged for the valid items in the [dataset](https://docs.apify.com/platform/storage/dataset) of the Actor run.
```

Comment:
`Technically, we validate the _results_ against a schema, and you are only charged for the valid items in the [dataset](https://docs.apify.com/platform/storage/dataset) of the Actor run.`

Check this with Jarda or Zenda, I don't think everything is validated just yet.